lighthouse-core/gather/gather-runner.js

```diff
@@ -85,12 +86,18 @@ class GatherRunner {
   static setupDriver(driver, options) {
     log.log('status', 'Initializing…');
     // Enable emulation if required.
-    return Promise.resolve(options.flags.mobile && driver.beginEmulation())
+    return driver.checkForMultipleTabsAttached(options.url)
+      .then(_ => options.flags.mobile && driver.beginEmulation())
       .then(_ => driver.enableRuntimeEvents())
       .then(_ => driver.cleanAndDisableBrowserCaches())
       .then(_ => driver.clearDataForOrigin(options.url));
   }
 
+  static disposeDriver(driver) {
+    log.log('status', 'Disconnecting from browser...');
+    driver.disconnect();
```

Comment:
should copy the
```js
// We dont need to hold up the reporting for the reload/disconnect,
// so we will not return a promise in here.
```
comment up here so it's clear why `driver.disconnect();` isn't being returned